configure.ac

```diff
@@ -3418,6 +3411,7 @@ if test x$host_win32 = xno; then
 	])
 
 	AC_CHECK_DECL(IN_EXCL_UNLINK, [AC_DEFINE(HAVE_IN_EXCL_UNLINK, 1, [IN_EXCL_UNLINK])], [], [[#include <sys/inotify.h>]])
+	AC_CHECK_DECL(TCP_KEEPALIVE, [AC_DEFINE(HAVE_TCP_H_TCP_KEEPALIVE, 1, [TCP_KEEPALIVE])], [], [[#include <netinet/tcp.h>]])
```

Comment:
What is this line regarding?

Reply:
ah yeah, it's not really needed. it's an additional define that they added on the corefx side that was not ported to our configure.ac